.env.development

```diff
@@ -0,0 +1,23 @@
+REACT_APP_ELASTIC_APM_SERVICE_VERSION=${ELASTIC_APM_SERVICE_VERSION}
+REACT_APP_ELASTIC_APM_SERVICE_NAME=${ELASTIC_APM_SERVICE_NAME}
+REACT_APP_ELASTIC_APM_JS_BASE_SERVICE_VERSION=${ELASTIC_APM_JS_BASE_SERVICE_VERSION}
+REACT_APP_ELASTIC_APM_SERVICE_VERSION=${ELASTIC_APM_SERVICE_VERSION}
+REACT_APP_ELASTIC_APM_JS_BASE_SERVER_URL=${ELASTIC_APM_JS_BASE_SERVER_URL}
+REACT_APP_ELASTIC_APM_SERVER_URL=${ELASTIC_APM_SERVER_URL}
+REACT_APP_ELASTIC_APM_ACTIVE=${ELASTIC_APM_ACTIVE}
+REACT_APP_ELASTIC_APM_BREAKDOWN_METRICS=${ELASTIC_APM_BREAKDOWN_METRICS}
+REACT_APP_ELASTIC_APM_CENTRAL_CONFIG=${ELASTIC_APM_CENTRAL_CONFIG}
+REACT_APP_ELASTIC_APM_DISABLE_INSTRUMENT=${ELASTIC_APM_DISABLE_INSTRUMENT}
+REACT_APP_ELASTIC_APM_DISTRIBUTED_TRACING=${ELASTIC_APM_DISTRIBUTED_TRACING}
+REACT_APP_ELASTIC_APM_DISTRIBUTED_TRACING_ORIGINS=${ELASTIC_APM_DISTRIBUTED_TRACING_ORIGINS}
+REACT_APP_ELASTIC_APM_ENVIRONMENT=${ELASTIC_APM_ENVIRONMENT}
+REACT_APP_ELASTIC_APM_ERROR_THROTTLE_INTERVAL=${ELASTIC_APM_ERROR_THROTTLE_INTERVAL}
+REACT_APP_ELASTIC_APM_ERROR_THROTTLE_LIMIT=${ELASTIC_APM_ERROR_THROTTLE_LIMIT}
```

Comment:
I don't recognize these two. They aren't in https://www.elastic.co/guide/en/apm/agent/rum-js/current/configuration.html